modal/io_streams.py

```diff
@@ -17,7 +19,7 @@
 
 async def _sandbox_logs_iterator(
     sandbox_id: str, file_descriptor: int, last_entry_id: Optional[str], client: _Client
-) -> AsyncIterator[Tuple[Optional[api_pb2.TaskLogs], str]]:
+) -> AsyncIterator[Tuple[Optional[str], str]]:
```

Comment:
The return type here should preferably use the `AsyncGenerator` generic which is more specific (letting the type checking system know that you can call `.aclosing()` etc. on it). I see it was AsyncIterator even before this change though, but could be nice to fix